tst/testinstall/mgmring.tst

```diff
@@ -18,8 +18,8 @@ gap> IsGroupRing( rm );
 true
 gap> centre:= Centre( rm );
 <algebra-with-one of dimension 3 over GF(3)>
-gap> GeneratorsOfAlgebra( centre );
-[ (Z(3)^0)*(), (Z(3)^0)*(), (Z(3)^0)*(2,3)+(Z(3)^0)*(1,2)+(Z(3)^0)*(1,3), 
+gap> AsSet( GeneratorsOfAlgebra( centre ) );
+[ (Z(3)^0)*(), (Z(3)^0)*(2,3)+(Z(3)^0)*(1,2)+(Z(3)^0)*(1,3), 
```

Comment:
Two different methods are called here depending on which packages (and thus implications are loaded), but both really are fine.